src/components/component-playground.jsx

```diff
@@ -180,13 +181,44 @@ module.exports = React.createClass({
     </ul>;
   },
 
-  _renderContentFrame: function() {
-    return <div ref="contentFrame" className={this._getContentFrameClasses()}>
+  _renderPreview: function() {
+    return (
```

Comment:
I don't like the `(`. Put the `<div ref="previewContainer" className={this._getPreviewClasses()}>` after the return statement.